rewrite-groovy/src/main/java/org/openrewrite/groovy/tree/G.java

```diff
@@ -1020,4 +1020,109 @@ public G.Unary withOperator(JLeftPadded<G.Unary.Type> operator) {
             }
         }
     }
+
+    @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+    @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
+    @RequiredArgsConstructor
+    @AllArgsConstructor(access = AccessLevel.PRIVATE)
+    final class MultipleAssignmentDeclaration implements G, Statement {
+        @Nullable
+        @NonFinal
+        transient WeakReference<Padding> padding;
+
+        @With
+        @EqualsAndHashCode.Include
+        @Getter
+        UUID id;
+
+        @With
+        @Getter
+        Space prefix;
+
+        @With
+        @Getter
+        Markers markers;
+
+        @With
+        @Getter
+        List<Modifier> modifiers;
+
+        @With
+        @Nullable
+        @Getter
+        TypeTree typeExpression;
+
+
+        @Getter
+        @With
+        JLeftPadded<Expression> initializer;
+
+        JContainer<VariableDeclarations.NamedVariable> elements;
```

Comment:
Should this be `variables` (and associated methods)?

Reply:
I will update it